clients/python/README.md

```diff
@@ -0,0 +1,69 @@
+# Stencil Python client
+
+[![PyPI version](https://badge.fury.io/py/stencil-python-client.svg)](https://pypi.org/project/stencil-python-client)
+
+Stencil Python client package provides a store to lookup protobuf descriptors and options to keep the protobuf descriptors upto date.
+
+It has following features
+ - Deserialize protobuf messages directly by specifying protobuf message name
+ - Ability to refresh protobuf descriptors in specified intervals
+ - Support to download descriptors from multiple urls
+
+
+## Requirements
+
+ - Python 3.7+
+
+## Installation
+
+Use `pip`
+```
+pip3 install stencil-python-client
+```
+
+Then import the stencil package into your own code as mentioned below
+```python
+from raystack import stencil
+```
+
+## Usage
+
+### Creating a client
+
+```python
+from raystack import stencil
+
+url = "http://url/to/proto/descriptorset/file"
+client = stencil.Client(url)
+```
+
+### Creating a multiURLClient
+
+```python
+from raystack import stencil
+
+urls = ["http://urlA", "http://urlB"]
+client = stencil.MultiUrlClient(urls)
+```
+
+### Get Descriptor
+```python
+from raystack import stencil
+
+url = "http://url/to/proto/descriptorset/file"
+client = stencil.Client(url)
+client.get_descriptor("google.protobuf.DescriptorProto")
```

Comment:
We should be adding  how we use the descriptor object as well, since it wasn't quite straightforward to use IIRC

Reply:
the parse protobuf message section has the example, let me know if more details required